docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -6,6 +6,152 @@ description: "Review which adjustments must be made to migrate from Camunda 8.7.
 
 The following sections explain which adjustments must be made to migrate from Camunda 8.7.x to 8.8.x for each component.
 
+## Camunda Exporter
+
+In the past release, we’ve engineered a new Camunda Exporter, which brings the importer and archiving logic of web components (Tasklist and Operate) closer to our distributed platform (Zeebe).
+This allows us to simplify our installation, enable scalability for web apps, reduce the latency to show runtime and historical data, and reduce data duplication (resource consumption).
+
+![87-state](./img/87-state-refined.png)
+
+Our web applications have historically used importers and archivers to consume, aggregate, and archive historical data provided by our Elasticsearch (ES) or OpenSearch (OS) Exporter.
+With the introduction of the Camunda Exporter, there are some changes in the architecture users should be aware of.
+
+![brownfield](./img/target-brown-field-refined.png)
+
+We have introduced the Camunda Exporter to simplify things and get several benefits from it.
+
+For the migration/update phase, we still need to run the Operate and Tasklist importer and the Elasticsearch Exporter. This is to make sure that old
+data written in 8.7 to the Zeebe journal or already in the Zeebe Elasticsearch indices, gets imported into our new
+harmonized index schema.
+
+### Harmonized Indices
+
+The harmonized schema looks like the following:
+
+![harmoninzed-schema](./img/harmonized-indices-miro-v2.png)
+
+We were able to reduce the need for several indices and to make use of existing ones (Operate indices are used by Tasklist, etc.)
+
+### Notes for updating to 8.8
+
+Notes for updating to 8.7 related to the Camunda Exporter and harmonized indices.
+
+#### Users need to make sure that the exporter/import backlog is small
+
+As we mentioned above we need to import old data from 8.7 via the importers.
+The importers need to run first and complete before the Camunda Exporter can confidently start.
+
+The Importer is reading data, from existing Zeebe Elasticsearch indices, and as soon it detects data from newer versions,
+it marks itself as complete. The Camunda Exporter can detect this and will start exporting only if the Importer is done.
+
+:::important
+To reduce the potential of building a backlog,
+the user needs to make sure to drain the backlog queue as much as possible before the update.
+:::
+
+The backlog can be validated via the following metrics:
```

Comment:
I suggest to make it explicit:

`following metrics` => `following Prometheus metrics`